shipwire-grabber.py

```diff
@@ -19,7 +19,8 @@
 shipwire = Shipwire(
             os.environ['SHIPWIRE_USER'],
             os.environ['SHIPWIRE_PASSWORD'],
-            host='api.shipwire.com')
+            host='api.shipwire.com',
+            raise_on_errors=True)
```

Comment:
Woah, pretty weird to not raise on errors. I would have expected that to be the default. _Especially_ in a python lib!